pagerenderer/vue/ui.apps/src/main/content/jcr_root/apps/pagerender/vue/structure/page/explorer_dialog.json

```diff
@@ -15,6 +15,20 @@
     "label": "Title",
     "placeholder": "page title"
   },
+  {
+    "type": "material-select",
+    "label": "lang=",
```

Comment:
should we really use lang= as the label for this field?

Reply:
I was thinking about a label like "Site Language"